package.json

```diff
@@ -54,17 +54,15 @@
 	},
 	"ava": {
 		"require": [
-			"@babel/register"
-		]
-	},
-	"babel": {
-		"presets": [
-			"@ava/stage-4"
+			"esm"
 		]
 	},
 	"xo": {
 		"globals": [
 			"atom"
-		]
+		],
+		"rules": {
+			"unicorn/prefer-optional-catch-binding": "off"
```

Comment:
I'm unsure why this is an issue... The node versions for CI and atom matchup. But for some reason CI fails when it attempts to parse `} catch {`